lib/web/apiserver.go

```diff
@@ -869,6 +869,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	// Kube access handlers.
 	h.GET("/webapi/sites/:site/kubernetes", h.WithClusterAuth(h.clusterKubesGet))
 	h.GET("/webapi/sites/:site/pods", h.WithClusterAuth(h.clusterKubePodsGet))
+	h.GET("/webapi/sites/:site/kubernetes/resources", h.WithClusterAuth(h.clusterKubeResourcesGet))
```

Comment:
i added a generic endpoint to allow requesting all sub resources @tigrato

Reply:
You can drop the `h.GET("/webapi/sites/:site/pods", h.WithClusterAuth(h.clusterKubePodsGet))` endpoint. 
It was never used and will be replaced by `h.GET("/webapi/sites/:site/kubernetes/resources", h.WithClusterAuth(h.clusterKubeResourcesGet))`